meraki/aio/__init__.py

```diff
@@ -90,6 +92,7 @@ def __init__(
             be_geo_id=BE_GEO_ID,
             caller=MERAKI_PYTHON_SDK_CALLER,
             use_iterator_for_get_pages=False,
+            inherit_logging_config=INHERIT_LOGGING_CONFIG
```

Comment:
This can be at the same order in the kwarg list in both __init__ files. Will fix.